src/main/java/com/khoros/TweetME/CreateTweet.java

```diff
@@ -15,7 +15,7 @@ public static Twitter getTwitterinstance() {
 
     public String createTweet(String tweet) throws TwitterException {
         Twitter twitter = getTwitterinstance();
```

Comment:
Please see the usage of singelton class 

 "https://www.baeldung.com/java-singleton"